### PR TITLE
[SI-607] Adds cypress test for Categoriser Home page

### PR DESCRIPTION
**Pull request by a6software**
Intended to be a Cypress replica of https://github.com/ministryofjustice/offender-categorisation/blob/main/integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/HomePageSpecification.groovy#L20-L69

### CODE REVIEW

File: integration_tests/mockApis/elite2.ts [hunk 1]
```diff
@@ -645,6 +647,34 @@ const stubGetOffenderDetails = ({
   return Promise.all([stubBasicInfo(), stubSentenceDetail(), stubSentenceTerms(), stubMainOffence()])
 }
 
+const stubGetOffenderDetailsBasic = ({
```
Comment (timothy-ness) [inline integration_tests/mockApis/elite2.ts:650]: Could you call stubGetOffenderDetails here with basic parameter set to true? The json body here overlaps with that functions, if basic is true should it be this smaller payload?

**Review by timothy-ness — DISMISSED** [review]

File: integration_tests/mockApis/elite2.ts (hunk 1 above)
Comment (a6software) [reply integration_tests/mockApis/elite2.ts]: implemented as suggested

File: integration_tests/mockApis/allocationManager.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export default {
     stubFor({
       request: {
         method: 'GET',
-        urlPattern: '/api/allocation/\\w+',
+        urlPattern: '/allocation-manager/api/allocation/\\w+',
```
Comment (a6software) [inline integration_tests/mockApis/allocationManager.ts:24]: the path was wrong here, which is why Engelbert Humperdinck now needed adding anywhere POM is displayed - it should always have displayed, but the mock was not properly setup.

**Review by timothy-ness — APPROVED** [review]

**Comment by a6software** [comment]
thanks

**Pull request merged**